custom/icds_reports/migrations/sql_templates/create_datasource_views.sql

```diff
@@ -452,6 +452,59 @@ CREATE VIEW agg_awc_daily_view AS
         ("awc_location"."doc_id" = "agg_awc"."awc_id")
     );
 
+CREATE TABLE IF NOT EXISTS public."config_report_icds-cas_static-child_health_cases_a46c129f"
```

Comment:
Thanks for the explanation. I'm not sure hwat the best solution to this is at the moment